### PR TITLE
addition of hindi for maintainer guide

### PR DESCRIPTION
**Pull request by VIDYA1980**
This PR adds a Hindi translation of the Maintainer Guide, enhancing accessibility and inclusivity for Hindi-speaking contributors and maintainers.
### Checklist
-  Added Hindi translations
Thank you for your consideration!

### CODE REVIEW

**Review by TenTraicion — CHANGES_REQUESTED** [review]
@VIDYA1980, Hello there, thanks for your contribution. but this PR doesn't follow our project rule properly. 
- The file naming isn't according to our rules.
- the codes mentioned in the guide shouldn't be translated
- some links are broken
- the file isn't properly translated according to our main [Maintainers Guide](https://github.com/Syknapse/Contribute-To-This-Project/blob/master/maintainer_guide.md)
- The translation isn't mentioned in the main [Maintainers Guide](https://github.com/Syknapse/Contribute-To-This-Project/blob/master/maintainer_guide.md)

Note: Kindly follow the rules mentioned in [Translation Guide](https://github.com/Syknapse/Contribute-To-This-Project/blob/master/translations/translation.md) properly. and make required changes. Thanks

File: translations/translation.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ All Translations regarding Project Documentations are welcome. There are certain
 | [English](../maintainer_guide.md)  | [Bangla](maintainer_guide/maintainer_guide.ben.md) | [German](maintainer_guide/maintainer_guide.ger.md) | [Italian](maintainer_guide/maintainer_guide.ita.md) | [Japanese](maintainer_guide/maintainer_guide.jpn.md) |
 | :---: | :---: | :---: | :---: | :---: |
 | [Portuguese](maintainer_guide/maintainer_guide.por.md) | [Ukrainian](maintainer_guide/maintainer_guide.ukr.md) |
+[Hindi](maintainer_guide/maintainer_guide_hin.md)
```
Comment (TenTraicion) [inline translations/translation.md:34]:
```suggestion
| [English](../maintainer_guide.md)  | [Bangla](maintainer_guide/maintainer_guide.ben.md) | [German](maintainer_guide/maintainer_guide.ger.md) | [Hindi](maintainer_guide/maintainer_guide_hin.md) | [Italian](maintainer_guide/maintainer_guide.ita.md) |
| :---: | :---: | :---: | :---: | :---: |
| [Japanese](maintainer_guide/maintainer_guide.jpn.md) | [Portuguese](maintainer_guide/maintainer_guide.por.md) | [Ukrainian](maintainer_guide/maintainer_guide.ukr.md) |
```

File: maintainer_guide.md [hunk 2]
```diff
@@ -8,8 +8,7 @@ This tutorial is also available in [other languages](translations/translation.md
 
 | [English](maintainer_guide.md)   | [Bangla](translations/maintainer_guide/maintainer_guide.ben.md) | [German](translations/maintainer_guide/maintainer_guide.ger.md) | [Italian](translations/maintainer_guide/maintainer_guide.ita.md) | [Japanese](translations/maintainer_guide/maintainer_guide.jpn.md) |
 | :---: | :---: | :---: | :---: | :---: |
-| [Portuguese](translations/maintainer_guide/maintainer_guide.por.md) | [Ukrainian](/translations/maintainer_guide/maintainer_guide.ukr.md) |
-
+| [Portuguese](translations/maintainer_guide/maintainer_guide.por.md) | [Ukrainian](/translations/maintainer_guide/maintainer_guide.ukr.md) |[Hindi](translations/maintainer_guide/maintainer_guide.hin.md)|
```
Comment (TenTraicion) [inline maintainer_guide.md:11]:
```suggestion
| [English](../maintainer_guide.md)  | [Bangla](maintainer_guide/maintainer_guide.ben.md) | [German](maintainer_guide/maintainer_guide.ger.md) | [Hindi](maintainer_guide/maintainer_guide_hin.md) | [Italian](maintainer_guide/maintainer_guide.ita.md) |
| :---: | :---: | :---: | :---: | :---: |
| [Japanese](maintainer_guide/maintainer_guide.jpn.md) | [Portuguese](maintainer_guide/maintainer_guide.por.md) | [Ukrainian](maintainer_guide/maintainer_guide.ukr.md) |
```

File: translations/maintainer_guide/maintainer_guide_hin.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
+# अनुरक्षक की मार्गदर्शिका
+यह मार्गदर्शिका उन लोगों के लिए है जो समुदाय के लिए परियोजना के प्रबंधन और रखरखाव में सहायता के लिए अनुरक्षक के रूप में परियोजना में शामिल होना चाहते हैं। (यह **पहली बार योगदान करने वालों** के लिए मार्गदर्शिका नहीं है)
+## उद्देश्य
```
Comment (TenTraicion) [inline translations/maintainer_guide/maintainer_guide_hin.md:3]: The Translate section is missing. You don't need to add the table, but add the translation section with other properties.

File: translations/maintainer_guide/maintainer_guide_hin.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+```
+इसे फ़ाइल को फ़ॉर्मेट करने का प्रयास करना चाहिए और यदि ऐसा नहीं हो पाता है तो यह आपको त्रुटियाँ दिखाएगा। कभी-कभी कोई गुम हुआ क्लोजिंग टैग या टूटा हुआ HTML गलती से मर्ज हो जाता है और यह उसे पहचानने और ठीक करने का एक अच्छा तरीका है।
+
+यदि आप कभी भी संदेह में हों, तो आप हमेशा पीआर में मेरा या अन्य अनुरक्षकों का उल्लेख कर सकते हैं या मुझे [Twitter] पर डीएम कर सकते हैं। (https://twitter.com/Syknapse).
+
+## हमसे जुड़ें
+इस परियोजना को एक साथ आगे बढ़ाने में मदद करने के लिए हमसे जुड़ें। [Twitter] पर मुझसे संपर्क करें
+(https://twitter.com/Syknapse) 
+और मुझे अपना GitHub उपयोगकर्ता नाम भेजें ताकि मैं आपको जोड़ सकूं। आप नीचे दिए गए बटन पर क्लिक करके भी हमारे Discord समुदाय में शामिल हो सकते हैं:
+[![Discord](https://badgen.net/discord/online-members/tWkvS4ueVF?label=Join%20Our%20Discord%20Server&icon=discord)](https://discord.gg/tWkvS4ueVF 'Join our Discord server!')
```
Comment (TenTraicion) [inline translations/maintainer_guide/maintainer_guide_hin.md:54]: there should be a line break before the badge

**Review by TenTraicion — CHANGES_REQUESTED** [review]
@VIDYA1980 the file name should be: `maintainer_guide.hin.md`
also it needs some changes mentioned below:

**Comment by VIDYA1980** [comment]
hey i have modified all the changes as requested kindly check and let me
know if anything else need to be changed ,I am beginner sorry for the
mistakes i have done previously

**Comment by TenTraicion** [comment]
Closing: The changes requested has not been fixed and it's been over 7 Days

Note: Kindly read the proper guidelines in our [Translation Guide](https://github.com/Syknapse/Contribute-To-This-Project/blob/master/translations/README.md) also it's related to the Issue: #2881 

Thanks and Regards

**Pull request closed without merging**